overlord/devicestate/handlers_install.go

```diff
@@ -92,3 +119,29 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 
 	return nil
 }
+
+// TODO:UC20: set to real TPM availability check function
+var checkTPMAvailability = func() error {
+	return nil
+}
+
+func checkEncryption(model *asserts.Model) (res bool, err error) {
```

Comment:
I was confused by the meaning of the function. I was wondering if it should be given a clearer name or a docstring at least.

Reply:
Adding a note in PR #7999 which should land before this one.